Test/NotCSharp/Python/EASY-SortArrayByParityII.py

```diff
@@ -0,0 +1,17 @@
+#https://leetcode.com/problems/sort-array-by-parity-ii/submissions/
+
+class Solution:
+    def sortArrayByParityII(self, A: List[int]) -> List[int]:
+        lst = [0]*len(A)
+        m = 0
```

Comment:
please fix naming of variables. Readability is  very important

Reply:
also I believe this exists:
https://github.com/aus2101/Test/blob/master/Test/NotCSharp/Python/SortArrayByParity.py

Reply:
Sorry for not checking. Removed it from the PR.